src/routeList.js

```diff
@@ -0,0 +1,29 @@
+// import React from 'react';
+import Order from './views/Order'
+import CustomerMap from './views/Map/CustomerMap'
+// import AgentMAp from './views/Map/AgentMAp'
+import Summary from './views/Order/Summary'
+import Address from './views/Order/Address'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using exported name 'Address' as identifier for default export.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)